src/types/childrenbirthrecords.ts

```diff
@@ -0,0 +1,23 @@
+import { MyInfoField, StringValue, CodeAndDesc, MyInfoAttribute } from "./base"
+
+type MyInfoChildFull = {
+  birthcertno: StringValue
+  name: StringValue
+  hanyupinyinname: StringValue
+  aliasname: StringValue
+  hanyupinyinaliasname: StringValue
+  marriedname: StringValue
+  sex: CodeAndDesc
+  race: CodeAndDesc
+  secondaryrace: CodeAndDesc
+  dialect: CodeAndDesc
+  lifestatus: CodeAndDesc
+  dob: StringValue
+  tob: StringValue
+}
+
+export type MyInfoChildBirthRecordBelow21 = MyInfoChildFull
+export type MyInfoChildBirthRecordAbove21 = Pick<MyInfoChildFull, 'birthcertno'>
```

Comment:
do these need to be exported?

Reply:
i think it'll be helpful if a user wants to use it to type a function to distinguish between the two